fe/src/test/java/org/apache/doris/catalog/DynamicPartitionTableTest.java

```diff
@@ -293,6 +294,50 @@ public void testMissTimeUnit(@Injectable SystemInfoService systemInfoService,
         catalog.createTable(stmt);
     }
 
+    @Test
+    public void testMissSTART(@Injectable SystemInfoService systemInfoService,
```

Comment:
You could refer to `QueryPlanTest`, and create real table.

Reply:
ok, i will change it.

Reply:
done